aten/src/THC/THCGenerateBoolType.h

```diff
@@ -2,16 +2,17 @@
 #error "You must define THC_GENERIC_FILE before including THCGenerateBoolType.h"
 #endif
 
-// TODO: define accreal type once the correct value is known.
 #define scalar_t bool
 #define ureal bool
+#define accreal int64_t
```

Comment:
...I guess this works lol. I'm not sure there really is any "reasonable" choice of accreal for bool though. Which operator needed it?

Reply:
We discussed this same thing with TH and the result was: `// TODO: define accreal type once the correct value is known.`

Why is this different?